src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionDetails.cs

```diff
@@ -556,6 +556,18 @@ public string AzureAccountToken
             }
         }
 
+        public int ExpiresOn
```

Comment:
This should probably be a nullable int (`int?`) since it isn't going to exist in every request

Reply:
why do you way it won't exist in every request? It should be there

Reply:
This is the connection details. Not all connections have an azure token.

Reply:
oh I see, okay that makes sense